src/notebooks/controllers/notebookControllerManager.ts

```diff
@@ -528,33 +428,7 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
             return;
         }
 
-        const updatePromise = this.computePreferredNotebookController(document);
-        if (!this.isLocalLaunch) {
-            void updatePromise.finally(() => {
-                if (this.isLocalLaunch) {
-                    return;
-                }
-                if (this.failedToFetchRemoteKernels) {
```

Comment:
I removed this altogether for now. I assume you're doing this differently anyway.